cli/concept.md

```diff
@@ -0,0 +1,157 @@
+# Overview
+The Amplify cli enables front-end developers to easily set up the backend resources in the cloud. <br/>
```

Comment:
CLI should be in caps everywhere in the doc. Also use "AWS Amplify CLI toolchain" at least once in the beginning of every document then you can reduce to "Amplify CLI" after that.